databook/_toc.yml

```diff
@@ -16,74 +16,96 @@ chapters:
   - file: background/Ophys-ephys-comparison
   - file: background/Optotagging.md
   - file: background/Intrinsic-Signal-Imaging
-- file: visual-coding-2p/vc2p-background
+  #
+  # Just to draw attention to this needing stuff
+- file: physiology/physiology                             # This is a placeholder file that should get (NEEDS) actual content
+  # We have many spaces around it
+  # 
   sections:
-  - file: visual-coding-2p/vc2p-dataset
-  - file: visual-coding-2p/vc2p-session-data
-  - file: visual-coding-2p/vc2p-stimuli
-  - file: visual-coding-2p/vc2p-evoked-response
-  - file: visual-coding-2p/vc2p-cross-session-data
-  - file: visual-coding-2p/vc2p-analysis
-- file: visual-coding-np/vcnp
-  sections:
-  - file: visual-coding-np/vcnp-quickstart
-  - file: visual-coding-np/vcnp-data-access
-  - file: visual-coding-np/vcnp-units
-  - file: visual-coding-np/vcnp-receptive-fields
-  - file: visual-coding-np/vcnp-lfp
-  - file: visual-coding-np/vcnp-optotagging
-  - file: visual-coding-np/vcnp-session
-  - file: visual-coding-np/vcnp-cheatsheet
-  - file: visual-coding-np/vcnp-links
-- file: cell-type-lookup-table/ctlut-background
-  sections:
-  - file: cell-type-lookup-table/ctlut-session-data
-  - file: cell-type-lookup-table/ctlut-optotagging
-  - file: cell-type-lookup-table/ctlut-accessing-data
-  - file: cell-type-lookup-table/ctlut-identifying-tagged-units
-- file: visual-behavior/vb-background
-  sections:
-  - file: visual-behavior/VB-Behavior
-    title: Visual Behavior Task
+  - file: physiology/visual-coding-2p/vc2p-background
     sections:
-    - file: visual-behavior/VB-BehaviorSessionData
-      title: Behavior Session Data
-  - file: visual-behavior/VB-Ophys
+    - file: physiology/visual-coding-2p/vc2p-dataset
+    - file: physiology/visual-coding-2p/vc2p-session-data
+    - file: physiology/visual-coding-2p/vc2p-stimuli
+    - file: physiology/visual-coding-2p/vc2p-evoked-response
+    - file: physiology/visual-coding-2p/vc2p-cross-session-data
+    - file: physiology/visual-coding-2p/vc2p-analysis
+  # - file: visual-behavior/vb-background                 # This section needs to go somewhere, but not sure where yet; background stuff on visual behavior.
```

Comment:
this goes in the stimulus section

Reply:
Adjusted the file structure and updated ToC